src/services/exportService.ts

```
@@ -34,22 +34,55 @@ export interface ExportResult {
 }
 
 class ExportService {
-  private async getUserData(t: TranslationFunction): Promise<any[]> {
+  private async getUserData(t: TranslationFunction): Promise<User[]> {
```

ESLint

[error] 't' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)